code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -115,6 +115,8 @@
 			return id_card
 
 /mob/living/carbon/human/IsAdvancedToolUser()
+	if(istype(get_item_by_slot(SLOT_HEAD), /obj/item/clothing/head/helmet/monkeytranslator) && (ismonkeyman(src)))
+		return TRUE //even with the monkeylike trait, the uplifting device allows you to use these
```

Comment:
You should just give them the TRAIT_MONKEYLIKE when they are not wearing the hat

Reply:
I actually started with that- they have it plainly- but this futureproofs some shit. If i remkved the race check here its also work on disabled. This is just cleaner code that. I preferred